packages/math-input/package.json

```diff
@@ -80,4 +80,4 @@
         "react-transition-group": "^4.4.1"
     },
     "keywords": []
-}
+}
```

Comment:
Would you mind adding the trailing newline back in?  Some tools (especially CLI ones) misbehave if they're missing.